src/entry-server.js

```diff
@@ -72,6 +72,7 @@ export async function renderError({ err, req, res, buildManifest }) {
         helmet: Helmet.renderStatic(),
         rootHtml,
         buildManifest,
+        isomorphicState: { internalServerError: true },
```

Comment:
Instead of `internalServerError` what if we call it `renderedError: true` or perhaps just `error` and use https://github.com/sindresorhus/serialize-error?

Reply:
Nvm, leave it like this!